### DISCUSSION TITLE
[Support Request] openwrt上运行成功, 就是好奇几个点

### DISCUSSION THREAD

**Discussion by minkoonim** [daeuniverse/dae#662] [General]
### Checks

- [X] I have searched the existing issues
- [X] I have read the documentation
- [X] Is it your first time sumbitting an issue

### Support Request

1. 
我参考了 #79, 把DNSMASQ 默认"重定向"关了, 意思是要不是这样, DAE 和 DNSMASQ 互相争取抓取 DNS QUERY 的 UDP REQUEST嘛?

2.
我的 DNS.DAE 按官方配置模板设置, 
就把 google 和 ali 的 smartdns的两个口, 换成127.0.0.1:6053(替代谷歌), 127.0.0.1:6054(替代阿里),
```
upstream {
  overseas: 'udp://127.0.0.1:6053'
  china: 'udp://127.0.0.1:6054'
}

routing {
  request {
    qname(ext:'geosite.dat:category-ads') -> reject
    qname(ext:'geosite.dat:category-ads-all') -> reject
    qname(geosite:cn) -> china
    fallback: overseas
  }
  
  response {
    upstream(overseas) -> accept
    # Possibly polluted, re-lookup using overseas.
    ip(geoip:private) && !qname(geosite:cn) -> overseas
    fallback: accept
  }
}
```
我在这好奇的是
我按 1 修改INIT.D**之前**, 
试过dnsmasq的接听口改了0 (让他不要听53), 这时DNS QUERY 全失败.
反而让他听53口, 就能nslookup成功, 而且显示的 smartdns服务器名称.

3. 如果机场不支持IPV6, 而QUERY RESULT有IPV6 地址, 
DAE会觉得不能进, 然后发生 (tcp6) no alive dialer 是吗?

### Environment

- **Dae version (use `dae --version`)**: 
can't find dae, using daed-next,
daed-next version 2023-12-02-102cb56-core-2024-03-07
- **OS (e.g `cat /etc/os-release`)**: openwrt 23.05.2
- **Kernel (e.g. `uname -a`)**: 6.6.21
- **Others**: using daed-next

**Comment by dae-prow** [comment]
Thanks for opening this issue!
